packages/vitest/src/runtime/setup.ts

```diff
@@ -38,16 +38,17 @@ function setupDefines(defines: Record<string, any>) {
 }
 
 export function setupConsoleLogSpy() {
-  const stdoutBuffer: any[] = []
-  const stderrBuffer: any[] = []
-  let stdoutTime = 0
-  let stderrTime = 0
-  let timer: any
+  const stdoutBuffer = new Map<string, any[]>()
+  const stderrBuffer = new Map<string, any[]>()
+  const timers = new Map<string, { stdoutTime: number; stderrTime: number; timer: any }>()
+  const unknownTestId = '__vitest__unknown_test__'
```

Comment:
should move it to a common place to detect it on the ui

Reply:
uhmm, if sent to the client, logs shouldn't be displayed, and so there is no need to move it